code/game/gamemodes/miniantags/guardian/types/gaseous.dm

```diff
@@ -0,0 +1,82 @@
+//Gaseous
+/mob/living/simple_animal/hostile/guardian/gaseous
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	damage_transfer = 0.7
+	range = 7
+	playstyle_string = "As a <b>gaseous</b> type, you have only light damage resistance, but you can expel expelled_gasgas in an area. In addition, your punches cause sparks, and you make your summoner heat resistant."
+	magic_fluff_string = "..And draw the Atmospheric Technician, flooding the area with gas!"
+	tech_fluff_string = "Boot sequence complete. Atmospheric modules activated. Holoparasite swarm online."
+	bio_fluff_string = "Your scarab swarm finishes mutating and stirs to life, capable of spewing out many gases."
+	/// Moles of gas being expelled.
+	var/moles_of_gas = null
+	///Linda flag for the expelled gas because we need to use special flags for it that are not readable in game well.
+	var/linda_flags = null
+	/// Possible gases to expel, with how much moles they create.
+	var/static/list/possible_gases = list(
+		"Oxygen" = 50,
+		"Nitrogen" = 750, //overpressurizing is hard!.
+		"N2O" = 15,
+		"CO2" = 50,
+		"Plasma" = 5,
+		"Agent B" = 5
```

Comment:
Agent B? Does that even do anything in our current codebase?

Reply:
Discussed on discord, more niche gas for sm sabatoge or non antag use if a atmos tech gets a guardian from a BSH